scripts/update_data_propub

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+source .env.secrets
+
+if [[ -z "$PROPUBLICA_API_KEY" ]]; then
+    echo "Error. Must ensure /scripts/.env.secrets file exists and contains your PROPUBLICA_API_KEY. See /scripts/.env.secrets.example for more details." 1>&2
+    exit 1
+fi
+
+echo "ProPublica API key found. Updating Congress (House + Senate) data...."
```

Comment:
You should use the `util/print.sh` functions to get the fun colors, makes the output easier to read.